lib/Chat/Changelog/Manager.php

```diff
@@ -135,6 +135,11 @@ public function getChangelogs(): array {
 			. $this->l->t('- Moderators can now ban accounts and guests to prevent them from rejoining a conversation') . "\n"
 			. $this->l->t('- Upcoming calls from linked calendar events and out-of-office replacements are now shown in conversations') . "\n"
 			. $this->l->t('- Calls can now be done in a federated way with other Talk servers (requires the High-performance backend)'),
+			$this->l->t('## New in Talk %s', ['20.1']) . "\n"
+			. $this->l->t('- Desktop client for Windows, macOS and Linux is now available: %s', ['https://nextcloud.com/talk-desktop-install']) . "\n"
+			. $this->l->t('- Summarize call recordings and chats with many unread messages (requires AI configuration in the server)') . "\n"
```

Comment:
```suggestion
			. $this->l->t('- Summarize call recordings and unread messages in chats with the Nextcloud Assistant') . "\n"
```

Reply:
I think just naming the Assistant makes it clear enough you need that for it to work.